app/controllers/admin/enterprises_controller.rb

```diff
@@ -184,6 +186,11 @@ def load_taxons
       @taxons = Spree::Taxon.order(:name)
     end
 
+    def load_roles
+      @enterprise_roles = EnterpriseRole.find_all_by_enterprise_id(@enterprise.id)
```

Comment:
There is an `enterprise_roles` association on Enterprise model. Perhaps we could access this data in the injecor via `@enterprise.enterprise_roles`, rather than adding this logic to the controller? Same goes for the notification user: the logic for finding the notification user belongs on the model, not in a controller.

Reply:
Any thoughts on this @Matt-Yorkley? Just trying to keep our controllers nice and skinny.